### DISCUSSION TITLE
Hypothesis Formulation Discussion

### DISCUSSION THREAD

**Discussion by Gersh305** [ecn310/course-project-zipcentercrime#11] [Ideas]
This is where we will work on formulating our hypothesis

**Comment by sortizh** [comment]
### Zip Center Crime
1. *Research Question:* How does the distance to the closest drug rehabilitation center relate to the amount of calls to 911?
2.  *Hypothesis:* As the distance from a drug treatment facility increases the number of 911 calls also increase.
3.  *Economic Theory:* A closer proximity to drug treatment centers increases access to drug treatment and mental health resources which decreases drug usage. This decrease in drug usage will also decrease the amount of crimes related to drugs and drug usage, thus reducing 911 calls.
4. *Casual Diagram:*
 ```mermaid
---
title: 
---
flowchart LR
A["Distance to Treatment Centers"] --(#45;)--> B["Access to Treatment"]
B --(#45;)--> C["Drug Usage"] --(#43;)--> D["Crimes Related to Drugs"]--(#43;)--> E["Number of 911 Calls"]
```

5. *How to test the hypothesis:* We will use location, specifically long. and lat. for 911 calls in comparison to the location of drug treatment. centers. We will use ArchGIS to make layers of both the drug treatment center locations and the 911 call locations. Using those layers, we will have ArchGIS create a dataset for us based on the location of 911 calls from each drug treatment center. With that new dataset, we will conduct descriptive analysis.

**Reply by kbuzard** [reply]
Good job! You're ready to dig into the details of how you will test your hypothesis. We'll get to hypothesis testing the week after next; for now, concentrate on getting the data in the form you'll need that relates distance to treatment center to the number of 911 calls.
